templates/workers.yaml.j2

```diff
@@ -43,6 +43,7 @@ spec:
         - secretRef:
             name: "celery-secrets"
         resources: {{ k8s_worker_container["resources"] | to_json }}
+{% if k8s_worker_beat_enabled %}
```

Comment:
Unfortunately I think this will just leave beat running if it was already running (so you'd have to know what to delete manually); an alternative might be to put beat in a separate file and use the present/absent ternary in `k8ts_templates`?

Reply:
Good catch! I'll try that idea.